File: easy/strings/isAnagram.py
import collections	
def isAnagram(s,t):
	n = len(s)
	if len(s) != len(t):
		return False
	hash = collections.defaultdict(int)
	for i in range(n):
		hash[s[i]] += 1
		hash[t[i]] -= 1
	for x in s:
		if hash[x]:
			return False

File: easy/strings/isAnagram.cpp

bool isAnagram(string s, string t){
	int n = s.size();
	if(s.size() != t.size())
		return false;
	unordered_map<char, int> m;
	for(int i = 0; i < n; ++i){
		m[s[i]]++;
		m[t[i]]--;
	}
	for(auto i : s){
		if(m[i] != 0)
			return false;
	}
	return true;	
}